EIPS/eip-7092.md

```diff
@@ -0,0 +1,412 @@
+}
+```
+
+## Rationale
+
+The financial bond standard is designed to represent fixed income assets, which reprensent a loan made by an investor to a borrower.
+The proposed design has been motivated by the necessity to tokenize fixed income assets, and to represent the bond token with same
+characteristics as in traditional finance. Keeping the same properties as in tradional finance is necessary for issuers and investors
+to move to tokenized bonds without major difficulties. The same structure used in tradional finace, i.e issuer-investment bank-investors
+can be used for the bond standard, in that case the investment bank intermediary may be replaced by smart contracts. In the case of
+institutional issuance, the smart contracts can be managed by the investment bank. Decentralized exchanges may also use the bond standard
+to list bonds, in that case, decentralized exchanges will be in charge of managing the smart contracts. Other entities may also create
+tokenized bonds by integrating this financial bond interface.
+
+The use of terminology such as `issueVolume` and `principalOf` instead of `totalSupply` and `balanceOf` respectively, as used in other standards like
+[ERC-20](./eip-20) is motivated by the need to use same the terminology as with traditional bond. Furthermore, by adding the `data` input parameter
+in all `transfer` functions to allow the transfer of additional data, the ERC-7092 SHOULD NOT be compatible with the ERC-20 standard, even if both
+the `principalOf` and `balanceOf` functions were used.
+
+Another reason that has motivated not to extend other standards like the ERC-20 is to allow token explorer platforms like etherscan to represent
+the ERC-7092 token not as a simple ERC-20 token, but as a new token showing some bonds characteristics like the `principal`, the `coupon`,
+the `denomination`, and the `maturity date`. Those information are very important for bondholders to know the return on the capital invested.
+
+### Total Supply
+
+We made the choice not to define the `totalSupply` function beacause it can be derived from both the `issue volume` and the `denomination`.
+However it is RECOMMENDED to define the `totalSupply` function in any contract that implements this standard. In that case,
+the `total supply` MUST be equal to the ratio of the `issue volume` and the `denomination`.
+
+```javascript
+    totalSupply = issueVolume / denomination
+```
+
+### Account Balance
+
+Because the balance of an account can be derived from both the `principal` and the `denomination`, we made the choice not to define the `balanceOf` function.
+However it is RECOMMENDED to define the `balanceOf` function in any contract that implements this standard. In that case,
+the `balance` of an account MUST be equal to the ratio of the `principal` of that account and the `denomination`.
+
+```javascript
+    balanceOf(account) = principalOf(account) / denomination
+```
+
+### Backwards compatibility
```

Comment:
```suggestion
## Backwards compatibility
```

Reply:
## Backwards compatibility

doesn't pass the EIP Validator test. This is probably due to the fact that this is not defined in the template and should then be defined as a sub-sub-section.

For this reason, I will keep it as defined previously, i.e ## Backwards compatibility for the test to pass.

Reply:
Ah you need to capitalize "Compatibility". Using `###` is incorrect.